src/components/profile-view/profile-view.jsx

```diff
@@ -0,0 +1,137 @@
+import React from "react";
+import React, { useState, useEffect } from "react";
+import { Row, Col, Button, Form } from "react-bootstrap";
+
+import PropTypes from "prop-types";
+import { Card, Image } from "react-bootstrap";
+import { FavoriteMovies } from "./user-favoritemovie";
+import { UpdateUser } from "./user-infoupdate";
+import { UserInfo } from "./user-info";
+
+export const ProfileView = ({ token, user, movies, onSubmit }) => {
+    const storedUser = JSON.parse(localStorage.getItem("user"));
+
+    const [username, setUsername] = useState(user.UserName);
```

Comment:
You will have to be consistent with case when calling variable names, always check the source for what that should be.

You have called `user.UserName` But what you have in your localStorage is `user.Username`.

This is why you can't see the username and email in your profile view.